### DISCUSSION TITLE
FileUpload custom style not working in version 10.2.1

### DISCUSSION THREAD

**Discussion by tom-ramesh** [primefaces/community#1826] [PrimeReact] [answered]
I'm trying to get custom styling for the File Upload component using @emotion/styled. But it's not working with version 10.2.1. Tried the same on newer version 10.6.3 and its working without any issues . The upload component is a needed asap and the version upgrade is not preferred now as the application is live. 
Adding sample [StackBlitz ](https://stackblitz.com/edit/ragzir-aouhz2?file=src%2FFileUpload.tsx)code.

**Comment by nitrogenous** [comment, marked as answer]
We regret to inform you that we are unable to provide partial updates for the older versions at this time. Our team is currently focused on the development of the next-generation PrimeReact. As a result, the implementation of new features and components has been temporarily halted. We recommend upgrading to the newer versions if possible.
